spec/requests/users_spec.rb

```diff
@@ -18,11 +21,33 @@
       get user_path(user.id)
       expect(response).to have_http_status(:success)
     end
-    
+
     it 'assigns @user' do
       user = User.create!(name: 'Jhonny', age: 59, bio: 'I love axes')
       get user_path(user.id)
       expect(assigns(:user)).to eq(user)
     end
   end
+
+  describe 'GET /new' do
+    it 'returns http success' do
+      get users_new_path
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'POST /create' do
+    context 'when creating a user with valid params' do
+      it 'should create a new user' do
```

Comment:
```suggestion
      it 'create a new user' do
```

https://www.betterspecs.org/#should